src/server/game/ScriptMgr.h

```diff
@@ -50,6 +50,8 @@ class ModuleReference;
 class TestCase;
 class OutdoorPvP;
 
+#define VISIBLE_RANGE 166.0f // MAX visible range (size of grid)
```

Comment:
That's a very old leftover from TC, I'd rather not import it and use proper visibility functions. (I see there is only 3 occurences of usage in TC)